src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -42,8 +43,12 @@
 	 */
 	public static final String KUBERNETES_DEPLOYMENT_NODE_SELECTOR = "spring.cloud.deployer.kubernetes.deployment.nodeSelector";
 
+	/**
+	 * The maximum concurrent tasks allowed for this platform instance.
+	 */
 	private int maximumConcurrentTasks = 20;
 
+	@NestedConfigurationProperty
```

Comment:
I've never been a big fan of exposing this, but understand there was a use-case for more "advanced users" to be able to tweak some knobs here. Understood its in the docs, people can see it etc, but do we really want to put these options out there in the UI, publishing them even more? Maybe possible just to ignore this? Maybe a question for @sabbyanandan

Reply:
We talked about it yday. Most of the Fabric8 properties don't have an effect when there's only one platform, because Fabric8 properties get automatically bootstrapped in SCDF/Skipper when they get provisioned on K8s, which is much earlier in the lifecycle. And it is not overridable when deploying streams/tasks. 

However, when we have >1 platform, we will have >1 connection override possibilities also, so that's when someone can deploy stream/tasks with a different set of Fabric8 overrides. 

All that said, we thought of blacklisting the properties that don't apply as overrides, and likewise, it shouldn't show up in the dropdown in SCDF UI. I suggested blacklisting everything under Fabric8 (most of it are connection/auth props), but @markpollack wanted to check to see what can still apply, and allow them to be in the dropdown.

The blacklisting thing would be outside of the scope of this PR, though, a separate one possibly.

Reply:
got it - ok. yah +1 to black/white list of those properties or maybe some wrapped around them like `namespace` - just feels awkward to expose the underlying fabric8 lib